roles/core/bluebanquise/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Gathering OS specific variables
+- name: include_vars ░ Gather OS specific variables
```

Comment:
Is it expected to have '░' here while package and service tasks use '█' separator?

Reply:
I've noticed afterward this is for all the tasks with tag internal, so seems expected.